projects/storefrontapp-e2e-cypress/cypress/helpers/site-context-selector.ts

```diff
@@ -95,7 +95,7 @@ export function addressBookNextStep() {
     'getDeliveryPage'
   );
 
-  cy.get('cx-shipping-address .btn-primary').click({ force: true });
+  cy.get('cx-shipping-address .btn-primary').click();
```

Comment:
I don't think we should remove it, because the actionable state wouldn't cause any problem. It causes more problem at times now since it says the button is disabled at times  locally

Reply:
I believe the main issue is relating to our checkout failing at odd times because of it loading for a long time in the server making each step fail during the duration of the test. I believe Jimmy is working on `checkout-as-persistent-user` which tackles this issue, but we were having trouble with it as well. I think this is another big issue in our current flaky tests. We need to be able to pass the checkout process in cypress without it failing at odd times due to the server

Reply:
Actually e2e tests should resemble the way user can use application. And clicking disabled button is kinda breaking that assumption. Additionally, these buttons prevent you from going to early to the next step and causing redirect from other step guards. Any time I see `force: true` it is a red flag for me. I understand the usage for clicking hidden elements, but not in case of disabled buttons.

And if the button is disabled locally, when it should then it should be fixed in the application, not overlooked by e2e tests.

I didn't notice very long load times in checkout, but the failed requests that were not retried (Jalo).